utils/symbol.c

```diff
@@ -1430,3 +1423,71 @@ void print_symtabs(struct symtabs *symtabs)
 		symbol_putname(sym, name);
 	}
 }
+
+static unsigned long kaddr;
+
+static unsigned long get_kaddr(char *str)
+{
+	char *pch;
+	unsigned long addr;
+
+	pch = strtok (str,"-");
+	addr = strtoul(pch, NULL, 16);
```

Comment:
I don't think the `strtok()` is needed, just calling `strtoul(str, NULL, 16)` should do the job IMHO.